magento/pwa-studio
# Move 5.0 docs out of drafts

## Description

This PR moves all 5.0-related documentation out of the drafts folder and into an appropriate folder.
It also adds them to a Table of Contents.

## Related Issue

Closes PWA-271

## Acceptance 

None needed. These have already been accepted when they were initially created.

### Verification Stakeholders

@dpatil-magento To verify build

### Specification



### Verification Steps

1. Navigate to the `pwa-devdocs` directory
2. Run linting on the migrated files:
    - `npm run lint src/peregrine/talons/index.md`
    - `npm run lint src/pwa-buildpack/reference/buildpack-cli/create-project/index.md`
    - `npm run lint src/pwa-buildpack/scaffolding/custom-project-templates/index.md`
    - `npm run lint src/pwa-buildpack/scaffolding/index.md`
3. Verify no linting errors on each file
4. Build the HTML preview for the docs: `npm run develop`
5. Navigate to the new pages:
    - `<url>/peregrine/talons/`
    - `<url>/pwa-buildpack/reference/buildpack-cli/create-project/`
    - `<url>/pwa-buildpack/scaffolding/custom-project-templates/`
    - `<url>/pwa-buildpack/scaffolding/`
6. Verify pages exist and they have entries in their section's Table of Contents

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
